apps/okreads/browser/src/app/app.component.html

```diff
@@ -6,7 +6,7 @@ <h1>okreads</h1>
         <div>
           <button
             data-testing="toggle-reading-list"
-            mat-button
+            mat-flat-button class="reading-list"
```

Comment:
Are you sure this makes it more accessible?

Reply:
Updated it.